airbyte-api-server/src/main/java/io/airbyte/api/server/repositories/clients/SyncDto.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.api.server.repositories.clients;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SyncDto {
```

Comment:
For reviewers: we will end up replacing this with something autogenerated through the openAPI spec